src/vm/hints/secp/signature.rs

```diff
@@ -14,7 +14,6 @@ use crate::{
 };
 use num_bigint::BigInt;
 use std::collections::HashMap;
-
```

Comment:
restore this file

Reply:
Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/370/commits/e1729c955aa513b232c48eea55283ba337d57b7b